data/test/sort_datasource_impl_test.dart

```diff
@@ -104,6 +106,20 @@ void main() {
           .sortFiles([documentSort1, documentSort2, documentSort3], sorter);
       expect(result, [documentSort3, documentSort2, documentSort1]);
     });
+
+    test(
+        'sortFiles should return success with list WorkGroupNode has been sorted modification date',
+            () async {
+          SharedPreferences.setMockInitialValues({});
+
+          final _sharedPreferences = await SharedPreferences.getInstance();
+
+          _sortDataSourceImpl = SortDataSourceImpl(_sharedPreferences);
```

Comment:
Why not putting this in a constructor since we use it in the whole file
```
          final _sharedPreferences = await SharedPreferences.getInstance();

          _sortDataSourceImpl = SortDataSourceImpl(_sharedPreferences);
```